src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Alerts can be created by users with admin permission or higher from **Project Settings > Alerts > New Alert**.
+
+There are two types of alerts: Issue Alerts and Metric Alerts. 
```

Comment:
Suggest rephrasing (there are/there is constructions lead to dead writing and also we could remove the passive voice in the previous sentence): "Sentry users with administrative permissions or higher can create one of two types of alerts: Metric Alerts or Issue Alerts. To confirm or set administrative permissions, see **Project Settings > Alerts > New Alert**.

Reply:
I'd also suggest we define the two types of alerts here briefly, so users (who we know may not read through the document) have a high-level of understanding and can dig into the doc for more.